README.md

```diff
@@ -7,46 +7,45 @@
+| [**Interpret-Text-Contrib**](https://github.com/interpretml/interpret-text-contrib) | Supporting a collection of interpretability technique for models trained on **text (NLP) data**|
 
-The Interpret Community extends [Interpret](https://github.com/interpretml/interpret), an open source python package from Microsoft Research for training interpretable models and helping to explain blackbox systems, with additional interpretability techniques and  utility functions to handle real-world datasets and workflows.
 
-This repository contains an SDK and Jupyter notebooks with examples to showcase its use.
+This repository contains Interpret-Community SDK and Jupyter notebooks with examples to showcase its use.
 
 # Contents
 
 - [Overview of Interpret-Community](#intro)
-- [Target Audience](#target)
 - [Getting Started](#getting-started)
 - [Supported Models](#models)
-- [Supported Explainers](#explainers)
+- [Supported Interpretability Techniques](#explainers)
 - [Use Interpret-Community](#Example)
 - [Contributing](#Contributing)
 - [Code of Conduct](#code)
 
 # <a name="intro"></a>
 
 # Overview of Interpret-Community
-Interpret-Community incorporates community developed interpretability techniques under one roof with a unified set of data structures and visualization. Users could experiment with different interpretability techniques and easily perform comparative analysis to evaluate their brand new explainers. Using these tools, one can explain machine learning models globally on all data, or locally on a specific prediction using the state-of-art technologies in an easy-to-use and scalable fashion. In particular, this open-source toolkit:
-1. Actively incorporates innovative interpretability techniques, and allows for further expansion by researchers and data scientists
-2. Creates a common API across the integrated libraries
-3. Applies optimizations to make it possible to run on real-world datasets at scale
-4. Provides improvements such as the capability to "reverse the feature engineering pipeline" to provide model insights in terms of the original raw features rather than engineered features
-5. Provides interactive and exploratory visualization to empower data scientists to gain significant insight into their data
-
- # <a name="target"></a>
+Interpret-Community is an extension of the [Interpret](https://github.com/interpretml/interpret) repository, incorporating further community developed and experimental interpretability techniques and functionalities for using machine learning interpretability in real world scenarios. Interpret-Community makes it very easy for users to extend [Interpret](https://github.com/interpretml/interpret), adding their experimental techniques (or functionalities), and perform comparative analysis to evaluate their brand new capabilities.
```

Comment:
In the second sentence, I'd recommend removing `easy` (it somehow makes the statement weaker, like we're trying to sell something)
`adding` -> `add`
`their brand new capabilities` -> `the brand new capabilities of these repos` (question - what specifically are we saying is brand new? At this point all of the techniques we're using have been out for at least a year)

Reply:
Thanks. Makes sense.